suite-native/accounts/src/components/AccountListItemInteractive.tsx

```diff
@@ -11,7 +14,10 @@ export const AccountListItemInteractive = ({
     account,
     onSelectAccount,
 }: AccountListItemInteractiveProps) => (
-    <TouchableOpacity onPress={() => onSelectAccount(account.key)}>
-        <AccountListItem key={account.key} account={account} />
-    </TouchableOpacity>
+    <Box>
+        <TouchableOpacity onPress={() => onSelectAccount(account.key)}>
+            <AccountListItem key={account.key} account={account} />
+        </TouchableOpacity>
+        {account.symbol === 'eth' && <TokenList accountKey={account.key} />}
```

Comment:
Please create some utility function in tokens package `(symbol) => symbol === 'eth'` so we can easily extend that check later.

Reply:
[0a1bb8e](https://github.com/trezor/trezor-suite/pull/7757/commits/0a1bb8e03a8d45650350886eca8a1f4fabdf455a)